frontend/src/components/VSkeleton/VGridSkeleton.vue

```diff
@@ -44,19 +44,22 @@ export default defineComponent({
     },
     numElems: {
       type: Number,
-      default: function () {
-        if (this.isForTab === "all") return 20
-        if (this.isForTab === "image") return 30
-        return 8
-      },
     },
   },
-  setup() {
+  setup(props: { numElems: any; isForTab: string; }) {
```

Comment:
```suggestion
  setup(props) {
```
The types of props are defined above, and we don't need to add any type information here.

Reply:
Made the suggested changes.
But still the CI test is failing @obulat